cloudigrade/util/redhatcloud/sources.py

```diff
@@ -182,6 +183,7 @@ def extract_ids_from_kafka_message(message, headers):
     return account_number, platform_id
 
 
+@cache_memoize(60)
```

Comment:
Should we consider pulling the 60 here out into the configuration file? Would make it possible to adjust via an env var, so if we ever have to change it we can just update the deployment, not build a new image.

Reply:
Yes, that makes sense, I'll take care of that. Will WIP this PR in the meantime. Thanks!